api/pkg/shared/resource/resource.go

```diff
@@ -215,6 +215,7 @@ func withResourceVersionDetails(db *gorm.DB) *gorm.DB {
 	return db.
 		Preload("Resource").
 		Preload("Resource.Catalog").
+		Preload("Resource.Category").
```

Comment:
WDYT: Ordering of resource's categories as well like resource's tags

Reply:
+1 Can we add this?

Reply:
Will add it